packages/components/src/Menu/Menu.tsx

```diff
@@ -13,6 +13,11 @@ import { Button } from "../Button";
 import { Typography } from "../Typography";
 import { Icon, IconNames } from "../Icon";
 
+const variation = {
+  startOrStop: () => ({ opacity: 0, y: window.innerWidth > 640 ? 10 : 50 }),
```

Comment:
Is this where the actual animation is being defined?

Reply:
@chris-at-jobber  It is.